src/core/issues-notes.js

```diff
@@ -10,33 +10,31 @@
 // numbered to avoid involuntary clashes.
 // If the configuration has issueBase set to a non-empty string, and issues are
 // manually numbered, a link to the issue is created using issueBase and the issue number
+import { addId, fetchAndCache, parents } from "./utils";
 import css from "../deps/text!core/css/issues-notes.css";
-import { fetchAndCache } from "./utils";
 import hyperHTML from "../deps/hyperhtml";
 import { pub } from "./pubsubhub";
 export const name = "core/issues-notes";
 
 const MAX_GITHUB_REQUESTS = 60;
 
 function handleIssues(ins, ghIssues, conf) {
-  const $ins = $(ins);
   const { issueBase, githubAPI } = conf;
   const hasDataNum = !!document.querySelector(".issue[data-number]");
   let issueNum = 0;
-  const $issueSummary = $(
-    "<div><h2>" + conf.l10n.issue_summary + "</h2><ul></ul></div>"
-  );
-  const $issueList = $issueSummary.find("ul");
-  $ins
-    .filter((i, issue) => issue.parentNode)
-    .each((i, inno) => {
-      const $inno = $(inno);
-      const isIssue = $inno.hasClass("issue");
-      const isWarning = $inno.hasClass("warning");
-      const isEdNote = $inno.hasClass("ednote");
-      const isFeatureAtRisk = $inno.hasClass("atrisk");
-      const isInline = $inno[0].localName === "span";
-      const dataNum = $inno.attr("data-number");
+  const issueSummary = hyperHTML`<div><h2>${
+    conf.l10n.issue_summary
+  }</h2><ul></ul></div>`;
+  const issueList = issueSummary.querySelector("ul");
+  [...ins]
+    .filter(issue => issue.parentElement)
```

Comment:
nit: We probably don't need this filter because any element cannot lack a parent. Not a blocker, though.

Reply:
Yup. Makes sense. I was just rewriting the old code which used it. 
Updated now.